tests/test_mathematical.py

```diff
@@ -1445,6 +1445,40 @@ def test_power_scalar(shape, dtype):
     assert_allclose(result, expected, rtol=1e-6)
 
 
+class TestNanToNum:
```

Comment:
please move new class before `class TestNextafter` to keep lexicographical order